v0.3/cruorg/components/input/input.scss

```diff
@@ -0,0 +1,139 @@
+// Branding imports
+@import '../../branding-imports';
+
+/* Input styles */
+
+$cru-input-color:                       $cru-color-gray-dark;
+$cru-input-border-width:                1px;
+$cru-input-border-style:                solid;
+$cru-input-border-color:                $cru-color-gray-dark;
+$cru-input-padding:                     $cru-spacer-base;
+$cru-input-border-radius:               ($cru-spacer-base / 2);
+$cru-input-box-sizing:                  border-box;
+$cru-input-width:                       100%;
+$cru-input-placeholder-color:           $cru-color-gray-dark;
+$cru-input-focus-outline:               none;
+$cru-input-focus-border:                none;
+$cru-input-focus-box-shadow:            0 $cru-spacer-base ($cru-spacer * 2) rgba(0, 0, 0, 0.08);
+$cru-input-focus-caret-color:           $cru-color-yellow;
+$cru-inpput-focus-placeholder-color:    transparent;
+$cru-input-disabled-border-width:       1px;
+$cru-input-disabled-border-style:       solid;
+$cru-input-disabled-border-color:       $cru-color-gray-light;
+$cru-input-disabled-color:              $cru-color-gray-light;
+$cru-input-disabled-font-weight:        $cru-font-weight-normal;
+$cru-input-invalid-border-color:        $cru-color-yellow;
+
+@mixin cru-form-input {
+    &__text {
+        color: $cru-input-color;
+        border-width: $cru-input-border-width;
+        border-style: $cru-input-border-style;
+        border-color: $cru-input-border-color;
+        border-radius: $cru-input-border-radius;
+        padding: $cru-input-padding;
+        box-sizing: $cru-input-box-sizing;
+        width: $cru-input-width;
+        &::placeholder {
+            color: $cru-input-placeholder-color;
+        }
+        &:focus {
+            outline: $cru-input-focus-outline;
+            border: $cru-input-focus-border;
+            box-shadow: $cru-input-focus-box-shadow;
+            caret-color: $cru-input-focus-caret-color;
+        }
+        &:focus::placeholder {
+            color: transparent;
+        }
+        &:disabled {
+            border-width: $cru-input-disabled-border-width;
+            border-style: $cru-input-disabled-border-style;
+            border-color: $cru-input-disabled-border-color;
+            color: $cru-input-disabled-color;
+            font-weight: $cru-input-disabled-font-weight;
+        }
+        &:invalid {
+            border-color: $cru-input-invalid-border-color;
+        }
+    }
+}
+
+@mixin cru-form-input-without-borders {
+    &__text {
+        border-top: none;
+        border-left: none;
+        border-right: none;
+        border-radius: 0;
+        &:focus {
+            box-shadow: none;
+            border-bottom: $cru-input-border-width $cru-input-border-style $cru-color-yellow;
+        }
+    }
+}
+
+@mixin cru-form-input-black {
```

Comment:
@sebamelzi Please change references to the styles used for dark backgrounds to "dark" instead of "black".